authui/src/authflowv2/components/layout.css

```diff
@@ -1,15 +1,33 @@
 @layer components {
+  :root {
+    --layout-flex-align-items: center;
```

Comment:
The alignment button is not only updating the card alignment, but it should affect all alignments (i.e. text alignment, content alignment).

We have added the `alignment-logo` `alignment-content` attributes for this purpose. You can search these strings in the code base.

And I prefer to define the layout alignments following the existing pattern.

```
.layout--default {
    :root[alignment-content="start"] & {
      @apply items-start;
    }

    :root[alignment-content="center"] & {
      @apply items-center;
    }

    :root[alignment-content="end"] & {
      @apply items-end;
    }
}
```

@fungc-io Please correct me if it is not the case.

Reply:
According to design, I don't think this case should use `alignment-content`. Design is only the card is aligned to either side

Reply:
<img width="1169" alt="Screenshot 2024-06-28 at 9 49 13 PM" src="https://github.com/authgear/authgear-server/assets/10144357/a5e8b35e-68ab-41de-bfb5-5d2e329cc6c6">
<img width="1152" alt="Screenshot 2024-06-28 at 9 49 19 PM" src="https://github.com/authgear/authgear-server/assets/10144357/b45af8cd-2d23-49c2-bfdd-e1e134c2c7d3">

Reply:
But I can stick with the existing pattern and introduce another flag `alignment-card`

Reply:
ok. I remembered there was a design on content alignment customization. Maybe I mixed up something else.